### DISCUSSION TITLE
How to add function call directly in Kotaemon UI or with Gradio Client?

### DISCUSSION THREAD

**Discussion by elvino11** [Cinnamon/kotaemon#465] [Q&A]
Actually I already created my own API with Gradio Client to access Kotaemon. I want to ask how about function call in Kotaemon? is it available or not?

**Comment by taprosoft** [comment]
@elvino11 can you elaborate what you mean by function call. How does it differ from Gradio Client with built-in Gradio endpoint from Kotaemon?
